sc_preprocessing/0_qc_notebooks/scanpy_analysis_synovium_bulk.ipynb

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
<p>Are these commented out on purpose? I think you normalized counts in the other notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>yes -- because these are bulk data, not single-cell. I do normalization later in the pipeline</p>